server/src/main/java/com/octopus/teamcity/opentelemetry/server/TeamCityBuildListener.java

```diff
@@ -65,19 +70,17 @@ public TeamCityBuildListener(EventDispatcher<BuildServerListener> buildServerLis
     @Override
     public void buildStarted(@NotNull SRunningBuild build) {
         if (buildListenerReady()) {
-            String buildId = getBuildId(build);
-            String buildName = getBuildName(build);
-            Loggers.SERVER.debug("OTEL_PLUGIN: Build started method triggered for " + buildName);
+            Loggers.SERVER.debug("OTEL_PLUGIN: Build started method triggered for " + getBuildName(build));
 
             String parentBuildId = getParentBuild(build);
             Span parentSpan = this.otelHelper.getParentSpan(parentBuildId);
-            Span span = this.otelHelper.createSpan(buildId, parentSpan);
-            Loggers.SERVER.info("OTEL_PLUGIN: Span created for " + buildName);
+            Span span = this.otelHelper.createSpan(getBuildId(build), parentSpan);
+            Loggers.SERVER.info("OTEL_PLUGIN: Span created for " + getBuildName(build));
```

Comment:
Consider making this (and the next one a debug)
(feel free to make that out of scope for this PR)

Reply:
easy change! Change do